src/screens/Notifications/index.js

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
/**
https://www.drupal.org/docs/develop/standards/javascript/javascript-api-documentation-and-comment-standards